qiskit/ml/circuit/library/raw_feature_vector.py

```diff
@@ -0,0 +1,160 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The raw feature vector circuit."""
+
+from typing import Set, List, Optional
+import numpy as np
+from qiskit.circuit import QuantumRegister, ParameterVector, ParameterExpression, Gate
+from qiskit.circuit.library import BlueprintCircuit
+
+
+class RawFeatureVector(BlueprintCircuit):
+    """The raw feature vector circuit."""
```

Comment:
Can we have a better description for the documentation so a user will have a better idea about the class - the prior class had more text describing things here.

Reply:
Added in 80532db